code-challenges.test.js

```diff
@@ -25,27 +25,69 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
+// describe ("divisibleByThree", () => {
+//     it(`takes in an object and determines if that number is divisable by 3 evenly or not`, () => {
+//         expect(divisibleByThree(object1)).toEqual("15 is divisible by three")
+//         expect(divisibleByThree(object2)).toEqual("0 is divisible by three")
+//         expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")      
+//     })
+// })
+//ReferenceError: divisableByThree is not defined
+
+// const object1 = { number: 15 }
+// // Expected output: "15 is divisible by three"
+// const object2 = { number: 0 }
+// // Expected output: "0 is divisible by three"
+// const object3 = { number: -7 }
+// // Expected output: "-7 is not divisible by three"
+
+// // b) Create the function that makes the test pass.
+
+// const divisibleByThree = (object) => {
+//     //1) create a function divisible by three and call the objects with their key values.
+//     const number = object.number // made a new variable "number" to call the objects and their key value.
+//         if(number % 3 === 0){ // write a conditional statement to determine if that object can be divisible by 3 using a modulo.
+//             return `${number} is divisible by three`
+//         } else  {
+//             return `${number} is not divisible by three`
+//         }
+// }
```

Comment:
I would clean the code up a bit. You could try putting the pseudo code comments directly above the line of code it is explaining, maintaining the indentation of the code. Other than that, looks great!